lib/susedistribution.pm

```diff
@@ -168,5 +166,55 @@ sub become_root() {
     send_key('ctrl-l');
 }
 
+# initialize the consoles needed during our tests
+sub activate_consoles {
+    my ($self) = @_;
+    if (get_var('BACKEND', 'qemu')) {
+        activate_console('install-shell',    'tty-console', {tty => 2});
+        activate_console('root-console',     'tty-console', {tty => 2});
+        activate_console('bernhard-console', 'tty-console', {tty => 4});
```

Comment:
of course we can have "user-console" and map "user" to $testapi::username, but I don't want options to select_console, because the console object needs to know it's state. And that state *does* depend on the user, so two different users are 2 different consoles.